tilelang/language/__init__.py

```
@@ -136,6 +136,15 @@
 )
 
 
+tfloat32 = "tfloat32"
+tf32 = tfloat32
+dtype_registry = {
+    "float32": float32,
```

Ruff (0.15.7)

[error] `float32` may be undefined, or defined from star imports

(F405)